app/rental_route.py

```diff
@@ -0,0 +1,100 @@
+from app import db
+from app.models.video import Video
+from app.models.customer import Customer
+from app.models.rental import Rental
+from flask import request, Blueprint, make_response, jsonify
+from datetime import datetime
+import requests
+import json
+
+
+# creates a new varible  to help create my new endpoint
+rental_bp = Blueprint("rentals", __name__, url_prefix="/rentals")
+# /rentals/check-out The @rental_bp.route is create from ^^^
+
+
+@rental_bp.route("/check-out", methods=["POST"], strict_slashes=False)
+# <check-out> would mean I have a parameter that will change since I dont want my parameter I dont add it to my function and I leave the bracket off
+def check_out_video():
+    if request.method == "POST":
+
+        request_body = request.get_json()
+
+        if "customer_id" not in request_body or "video_id" not in request_body:
+            return make_response({
+                "details": "Invaild data"
+            }), 400
+
+        customer_id = request_body["customer_id"]
+        if isinstance(customer_id, int) is not True:
+            return make_response({
+                "details": "Invaild data"
+            }), 400
+        customer = Customer.query.get(customer_id)
+        # Getting videos_checked_out_count from customer object/class with the .
+        customer.videos_checked_out_count += 1
+
+        video_id = request_body["video_id"]
+        if isinstance(video_id, int) is not True:
+            return make_response({
+                "details": "Invaild data"
+            }), 400
+        video = Video.query.get(video_id)
+
+        video.available_inventory -= 1
+        if video.available_inventory < 0:
+            return make_response({
+                "details": "Invaild data"
+            }), 400
```

Comment:
Hmm since these all use the same return statement and error message, maybe we could come up with a way to have only one return statement that can catch all of these conditions